src/isIsogram.test.js

```diff
@@ -3,7 +3,35 @@
 describe('isIsogram', () => {
   const { isIsogram } = require('./isIsogram');
 
-  it(`should be declared`, () => {
+  it('should be declared', () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for a word with no repeating letters', () => {
+    expect(isIsogram('playgrounds')).toBe(true);
+  });
+
+  it('should return false for a word with repeating letters', () => {
+    expect(isIsogram('look')).toBe(false);
+  });
+
+  it('should return false for a word with repeating letters', () => {
+    expect(isIsogram('Adam')).toBe(false);
```

Comment:
The test case for 'Adam' assumes that the `isIsogram` function is case-sensitive. If the function is intended to be case-insensitive, you should normalize the case of the input string before checking for isograms.